test/emmy/collection_test.cljc

```diff
@@ -83,7 +83,13 @@
       (is (every? v/zero? (v/zero-like (list 1 2 3)))
           "works with lists"))))
 
+(defrecord MyRecord [])
+
 (deftest map-tests
+  (testing "don't try and simplify records"
```

Comment:
I had initially found this issue by trying to print `emmy.pattern.match/failure` in Cider, which I route through `expression->stream`.